content/source/docs/enterprise/system-overview/security-model.html.md

```diff
@@ -0,0 +1,100 @@
+
+TFE allows you to restrict access to the metadata endpoint from Terraform operations, preventing workspaces from reading any data from the metadata service. This feature can be enabled via the “Restrict Terraform Build Worker Instance Metadata Access” setting under the “Advanced Configuration” section in the installer dashboard’s settings page, or via the [restrict_worker_metadata_access](../install/automating-the-installer.html#restrict_worker_metadata_access) setting in the application settings file.
+
+Unless you are relying on the [instance profile as a means of providing default credentials to TFE workspaces](../before-installing/index.html#aws-specific-configuration), we recommend enabling this setting to prevent Terraform operations from accessing the instance metadata endpoint. 
+
+#### Ensure Global Remote State Sharing is Disabled
+
+TFE allows administrators to enable [global remote state sharing](../admin/general.html#remote-state-sharing), a feature that allows any workspace to access the state versions of any other workspace within the same organization. We recommend disabling the feature and relying on [controlled remote state access](https://www.hashicorp.com/blog/announcing-controlled-remote-state-access-for-terraform-cloud-and-enterprise) if you need to share state between workspaces.
+
+#### Treat Support Bundles with Care
+
+Support bundles are used to share diagnostic information with HashiCorp support. Please note that support bundles may contain sensitive information from your Terraform Enterprise installation. You should consider these bundles sensitive; they should not be shared with untrusted parties and should be deleted as soon as possible.
+
+#### Regularly Update Terraform Enterprise
+
+Terraform Enterprise updates are released on a monthly cadence. Updates may contain additional security features or fixes for existing security vulnerabilities, so we recommend establishing a process for periodically updating your Terraform Enterprise installation.
+
+#### Subscribe to Terraform Enterprise Security Bulletins
+
+HashiCorp publishes security updates, which address security vulnerabilities in HashiCorp products, in the Security category of [HashiCorp Discuss](https://discuss.hashicorp.com/c/security/).
+
+We recommend that Terraform Enterprise administrators follow the [documented steps](https://discuss.hashicorp.com/t/about-hashicorp-security-updates/15330) to subscribe to email notifications or the RSS feed for Terraform Enterprise security updates.
```

Comment:
both infrastructure and site admins?

Reply:
Infrastructure admins - they are the ones responsible for updating Terraform Enterprise.